### DISCUSSION TITLE
Possibility of linking two tables without a foreign key

### DISCUSSION THREAD

**Discussion by jeromeperez** [mathesar-foundation/mathesar#3550] [Feature Ideas]
It would be interesting to have a feature in "Data explorer" that would allow you to link a field from one table to another, so that you could display data from both databases.
For example :
dat_blessure(indid, date_debut,date_end)
individual(id,nom)
indid is not declared as a foreign key but id is unique in individual
```
select id,name,start_date,end_date,description
from dat_blessure,individu
where indid = id;
```

**Comment by seancolsen** [comment]
Thanks for this example @jeromeperez

Out of curiosity, why would you not want to have a foreign key here?

Also, I'm wondering whether you would be satisfied if the Data Explorer required you to use raw SQL to construct queries across tables without foreign keys? Or would you see it being important for users to build queries like this using the visual query building UI?

**Reply by jeromeperez** [reply]
1. indid correspond to id of individuals and in this case i prefer to use foreign key to do the link between the 2 tables.
2. oh yes i would be satisfied if the Data Explorer required you to use raw SQL to construct queries but it should be important too for users to build queries like this using visual query builder (in a second time) ;-)

**Reply by seancolsen** [reply]
Ok cool, thanks @jeromeperez
